x-pack/plugins/security_solution/public/siem_migrations/rules/logic/use_get_migration_translation_stats.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { useAppToasts } from '../../../common/hooks/use_app_toasts';
+import { useGetMigrationTranslationStatsQuery } from '../api/hooks/use_get_migration_translation_stats_query';
+import * as i18n from './translations';
+
+export const useGetMigrationTranslationStats = (migrationId: string) => {
+  const { addError } = useAppToasts();
+
+  return useGetMigrationTranslationStatsQuery(migrationId, {
+    onError: (error) => {
+      addError(error, { title: i18n.GET_MIGRATION_TRANSLATION_STATS_FAILURE });
+    },
+  });
```

Comment:
I see that for every API request we have to implement 2 separate hooks. Why don't we add error handling to the same hook that does the `useQuery`? so we have everything in one hook. Or is there a reason to have them separate?

Reply:
No real reason, except of following rules management patterns :-) I think we do not need to overcomplicate things and can keep everything in one hook. I will fix that in a separate PR.